tests/testthat/test-bar_plot_emission_profile.R

```diff
@@ -40,7 +63,12 @@ test_that("returns correct benchmarks values for equal weight mode", {
 
 test_that("returns correct benchmarks values for best case mode", {
   data <- example_without_financial()
-  data <- prepare_bar_plot_emission_profile(data, benchmarks(), "best_case")
+  mode <- "best_case" |>
+    switch_mode_emission_profile()
+  data <- prepare_bar_plot_emission_profile(
+    data, benchmarks(), mode,
+    scenarios()[1], years()[1]
+  )
```

Comment:
I think the tidyverse style would say to either put everything in one line or each one in one line.